app/src/main/java/com/paypal/psix/services/UserSession.java

```diff
@@ -0,0 +1,74 @@
+package com.paypal.psix.services;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import com.facebook.FacebookRequestError;
+import com.facebook.Request;
+import com.facebook.Response;
+import com.facebook.Session;
+import com.facebook.model.GraphUser;
+import com.paypal.psix.PSixApplication;
+import com.paypal.psix.models.User;
+
+import bolts.Task;
+
+public class UserSession {
+
+    private static final String LOG_TAG = UserSession.class.getSimpleName();
+
+    private static final String CUR_USER_FBID_KEY = "cur_user_fbid";
+    private static final String CUR_USER_FIRST_NAME_KEY = "cur_user_first_name";
+    private static final String CUR_USER_LAST_NAME_KEY = "cur_user_last_name";
+
+    private static SharedPreferences getSharedPrefs() {
+        return PreferenceManager.getDefaultSharedPreferences(PSixApplication.getAppContext());
+    }
+
+    private UserSession(User currentUser) {
+        SharedPreferences.Editor editor = getSharedPrefs().edit();
+
+        editor.putString(CUR_USER_FBID_KEY, currentUser.fbUserId);
+        editor.putString(CUR_USER_FIRST_NAME_KEY, currentUser.firstName);
+        editor.putString(CUR_USER_LAST_NAME_KEY, currentUser.lastName);
+        editor.apply();
+
+        sessionInstance = this;
+    }
+
+    public static User getUser() {
+        String fbId = getSharedPrefs().getString(CUR_USER_FBID_KEY, null);
+        if (fbId != null) {
+            return new User(
+                    fbId,
+                    getSharedPrefs().getString(CUR_USER_FIRST_NAME_KEY, null),
+                    getSharedPrefs().getString(CUR_USER_LAST_NAME_KEY, null)
+            );
+        }
+
+        return null;
+    }
```

Comment:
what do you think off extending the api with these methods:
`isUserSignedIn`
`logout` (sets user to null).